addon/Feeds/LocalizationFeed.js

```diff
@@ -2,10 +2,17 @@ const {PrefsTarget} = require("sdk/preferences/event-target");
 const {findClosestLocale, getPreferedLocales} = require("sdk/l10n/locale");
 const Feed = require("../lib/Feed");
 const am = require("../../common/action-manager");
-
+const DEFAULT_LOCALE = "en-US";
 const STRINGS = require("../../data/locales/locales.json");
 const AVAILABLE_LOCALES = Object.keys(STRINGS);
-const DEFAULT_LOCALE = "en-US";
+
+function getLocalizedStrings(locale, allStrings = STRINGS) {
+  if (locale === DEFAULT_LOCALE) {
+    return allStrings[DEFAULT_LOCALE];
+  }
+  const strings = allStrings[locale];
+  return Object.assign({}, allStrings[DEFAULT_LOCALE], strings || {});
```

Comment:
maybe add a comment to this line that says this is in order to merge on missing strings?